.nvmrc

```diff
@@ -1 +1 @@
-14.15.0
+14.21.1
```

Comment:
Changing this will likely require changes in the github actions. Is there a strong reason to change?

Reply:
I'll make the change to the github actions. This is the change that gets highlighted first in [my notes]() because it pulls in the NodeJs version which contains ARM support via a V8 version:

> Apple ARM support in V8
>
> The current version of NodeJS set in .nvmrc has a version of V8 that does not support Apple's ARM family of processors. This caused issues where following the [README](https://github.com/aws/amazon-genomics-cli#readme) for building the CLI, failed at the [NVM step](https://github.com/aws/amazon-genomics-cli#node). Support for Apple ARM was back ported to NodeJs v14 LTS in 14.17.0.
>
> Additionaly, NodeJs compilation would fail if you had Python 1.10 or greater installed on your Mac.
>
> Fixes
>
> - Uses newer V8 support for Python 3.10+
> - Uses newer V8 support of Apple ARM processor family
> - Includes various NodeJS v14 LTS bug and security fixes